apps/crn-frontend/src/shared-research/export.ts

```diff
@@ -1,7 +1,8 @@
 import { EntityResponses, SearchEntityResponse } from '@asap-hub/algolia';
 import { ResearchOutputResponse } from '@asap-hub/model';
 import { isInternalUser } from '@asap-hub/validation';
-import { CsvFormatterStream, Row, format } from '@fast-csv/format';
+/* eslint-disable-next-line import/no-unresolved */
+import { stringify, Options, Stringifier } from 'csv-stringify/browser/esm';
```

Comment:
Eslint cant resolve this for the same reason we had to hack this module in the jest tests.

Reply:
We do support ESM modules not in the jest environment. I wanted to just import the CJS module that is included but couldn't.